tox.ini

```diff
@@ -2,10 +2,11 @@
 envlist=py2,py3
 
 [pytest]
-addopts=-m 'not functional'
+addopts=-m 'normal'
 markers=
+    normal: mark a test as a normal test.
```

Comment:
Perhaps `normal` is not the best expression. How about `standard`, or even `default`?

Reply:
OK, I prefer `standard`. I will change for that.